web/ui/react-app/src/QueryStatsView.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './QueryStatsView.css';
+
+export interface QueryStats{
+    loadTime: number;
+    resolution: number;
+    totalTimeSeries: number;
+}
+
+export default function QueryStatsView(props: QueryStats){
```

Comment:
I think this should looks like
```
const QueryStatsView = FC<QueryStats> = ({ loadTime, resolution, totalTimeSeries}) => { .... }
```

Reply:
Hmm, not see that form before and I can't get anything like it working? No point expanding them out as I created an interface to define them that's then shared too.
Going for:
`const QueryStatsView = (props: QueryStats) => {...}`

Reply:
Yep, That's fine too

Reply:
Figured it out, so done it the way you suggested.